### PR TITLE
[Components with more than 1 service] LPS-187673 Refactor ManagedServiceFactory components from multiple modules

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-187673](https://liferay.atlassian.net/browse/LPS-187673)

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:startup

**Comment by RenanUmV** [comment]
ci:test:lima

**Comment by RenanUmV** [comment]
ci:test:security

File: modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/configuration/admin/service/OAuth2InAssertionManagedServiceFactory.java [hunk 1]
```diff
@@ -41,16 +42,14 @@
...
 
 /**
  * @author Arthur Chan
  */
 @Component(
 	property = Constants.SERVICE_PID + "=com.liferay.oauth2.provider.rest.internal.configuration.OAuth2InAssertionConfiguration",
-	service = {
-		ManagedServiceFactory.class,
-		OAuth2InAssertionManagedServiceFactory.class
-	}
```
Comment (rafaprax) [inline modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/configuration/admin/service/OAuth2InAssertionManagedServiceFactory.java:53]: @RenanUmV @MrJohn1911 inline this class into the `LiferayJWTBearerGrantHandler`. Thx

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/configuration/admin/service/DLSizeLimitManagedServiceFactory.java [hunk 2]
```diff
@@ -48,9 +39,7 @@
 @Component(
 	configurationPid = "com.liferay.document.library.internal.configuration.DLSizeLimitConfiguration",
 	property = Constants.SERVICE_PID + "=com.liferay.document.library.internal.configuration.DLSizeLimitConfiguration.scoped",
-	service = {
-		DLSizeLimitManagedServiceFactory.class, ManagedServiceFactory.class
-	}
+	service = ManagedServiceFactory.class
 )
 public class DLSizeLimitManagedServiceFactory implements ManagedServiceFactory {
```
Comment (rafaprax) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/configuration/admin/service/DLSizeLimitManagedServiceFactory.java:45]: @RenanUmV @MrJohn1911, please rename to `DLSizeLimitHelper` and register the `ManagedServiceFactory` service dynamically into the activate method. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Pull request closed without merging**